packages/next/src/server/config-shared.ts

```diff
@@ -441,9 +442,14 @@ export interface ExperimentalConfig {
   useEarlyImport?: boolean
 
   /**
-   * Enables `fetch` requests to be proxied to the experimental text proxy server
+   * Enables `fetch` requests to be proxied to the experimental test proxy server
```

Comment:
To confirm: will running `next test` automatically set this value to `true`? Because it should :)

Reply:
I could not figure out how to do that unfortunately. Since we configure playwright to run the `next dev` command, it wasn't clear to me how we could possibly tell _that_ Next.js instance to enable this automatically. It may need a new internal env var or something. 

I punted on this feature to be added later.

Reply:
Created a linear ticket for this so we don't lose it.

Reply:
Ok to do in a separate PR. But, yeah, you got the right idea. An internal env would work. E.g. you'd have:

```
webServer: {
  cmd: 'NEXT_TEST_PROXY=true; next dev'
}
```

Or via a command-line arg:

```
cmd: 'next dev --test-proxy'
```